README.md

```diff
@@ -1,28 +1,37 @@
 # PSScriptAnalyzer Action
-Github action for running PSScriptAnalyzer and use ConvertToSARIF to generate a SARIF file.
 
-# Getting Started
-To run this action add the step below in your GitHub Action:
+Github action for running PSScriptAnalyzer and use ConvertToSARIF to generate a Static Analysis
```

Comment:
PSScriptAnalyzer here should be a link to the official module documentation at <https://learn.microsoft.com/powershell/module/psscriptanalyzer/invoke-scriptanalyzer> as it's the first time we're seeing it in the README.

I also think it could be a little more clear if rewritten to "This GitHub Action workflow scans your PowerShell scripts for defects and anti-patterns using `Invoke-ScriptAnalyzer` from the PSScriptAnalyzer module to generate…"